### PR TITLE
feat: target shortcuts and apple silicon

### PR DESCRIPTION
**Pull request by p7nov**
Explain how to use new arm64 simulator targets with existing [target shortcuts](https://kotlinlang.org/docs/mpp-share-on-platforms.html#use-target-shortcuts).

### CODE REVIEW

File: docs/topics/mpp/mpp-share-on-platforms.md [hunk 1]
```diff
@@ -111,6 +111,56 @@ kotlin {
...
+    // Add the ARM64 simulator target
+    iosSimulatorArm64()
+
+    // Set up dependencies between the source sets
+    sourceSets {
+        // ...
+        iosSimulatorArm64Main {
+            dependsOn(iosMain)
+        }
+        iosSimulatorArm64MainTest {
```
Comment (sellmair) [inline docs/topics/mpp/mpp-share-on-platforms.md:138]: iosSimulatorArm64MainTest -> iosSimulatorArm64Test 

Probably a typo ☺️

Comment (p7nov) [reply docs/topics/mpp/mpp-share-on-platforms.md:138]: Thanks! Fixed.

File: docs/topics/mpp/mpp-share-on-platforms.md [hunk 2]
```diff
@@ -111,6 +111,56 @@ kotlin {
 ```
 
 </tabs>
+
+#### Target shortcuts and ARM64 (Apple Silicon) simulators
+
+The target shortcuts `ios`, `watchos` and `tvos` don't include the simulator targets for ARM64 (Apple Silicon) platforms:
```
Comment (koshachy) [inline docs/topics/mpp/mpp-share-on-platforms.md:117]:
```suggestion
The target shortcuts `ios`, `watchos`, and `tvos` don't include the simulator targets for ARM64 (Apple Silicon) platforms:
```

Comment (koshachy) [reply docs/topics/mpp/mpp-share-on-platforms.md:117]: Hmm, I first put an serial comma, but now I'm not sure if it is needed here 🤔

**Review by koshachy — APPROVED** [review]
Pretty good!

File: docs/topics/mpp/mpp-share-on-platforms.md (hunk 2 above)
Comment (p7nov) [reply docs/topics/mpp/mpp-share-on-platforms.md:117]: I think you're right, we need one here.

**Pull request merged**